### PR TITLE
Add Horizon test Image

### PR DESCRIPTION
**Pull request by ashu-011**
This PR focuses on creating horizon test image with all the required resources for horizon tests to run . And cleanup the resources once the tests have run successfully .

### CODE REVIEW

File: container-images/tcib/base/os/horizontest/run_horizontest.sh [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+USER_NAME=horizontest
+PASSWORD=horizontest
+FLAVOR_NAME=m1.tiny  
+
+# assert mandatory variables have been set
+[[ -z ${ADMIN_USERNAME} ]] && echo "ADMIN_USERNAME not set" && exit 1
+[[ -z ${ADMIN_PASSWORD} ]] && echo "ADMIN_PASSWORD not set" && exit 1
+[[ -z ${DASHBOARD_URL} ]] && echo "DASHBOARD_URL not set" && exit 1
+[[ -z ${AUTH_URL} ]] && echo "AUTH_URL not set" && exit 1
+[[ -z ${REPO_URL} ]] && echo "REPO_URL not set" && exit 1
```
[the remark this answers is not in the document]
Comment (ashu-011) [reply container-images/tcib/base/os/horizontest/run_horizontest.sh]: Done

File: container-images/tcib/base/os/horizontest/run_horizontest.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+HORIZONTEST_DIR=/var/lib/horizontest
+HORIZON_LOGS_DIR_NAME=${HORIZON_LOGS_DIR_NAME:-"horizon"}
+IMAGE_FILE="/var/lib/horizontest/cirros-0.6.2-x86_64-disk.img"
+IMAGE_FILE_NAME=cirros-0.6.2-x86_64-disk
+IMAGE_URL=http://download.cirros-cloud.net/0.6.2/cirros-0.6.2-x86_64-disk.img
+PROJECT_NAME=horizontest
+USER_NAME=horizontest
+PASSWORD=horizontest
+FLAVOR_NAME=m1.tiny
```
[the remark this answers is not in the document]
Comment (ashu-011) [reply container-images/tcib/base/os/horizontest/run_horizontest.sh:13]: we don't need these parameters to be configurable , they are going to be pretty much constant

File: container-images/tcib/base/os/horizontest/run_horizontest.sh [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+# assert mandatory variables have been set
+[[ -z ${ADMIN_USERNAME} ]] && echo "ADMIN_USERNAME not set" && exit 1
+[[ -z ${ADMIN_PASSWORD} ]] && echo "ADMIN_PASSWORD not set" && exit 1
+[[ -z ${DASHBOARD_URL} ]] && echo "DASHBOARD_URL not set" && exit 1
+[[ -z ${AUTH_URL} ]] && echo "AUTH_URL not set" && exit 1
+[[ -z ${REPO_URL} ]] && echo "REPO_URL not set" && exit 1
+[[ -z ${HORIZON_REPO_BRANCH} ]] && HORIZON_REPO_BRANCH="rhos-18.0-trunk-patches"
+
+function create_custom_resources {
+    if ! openstack image list --os-cloud default -f value -c Name | grep ${IMAGE_FILE_NAME} >/dev/null; then
```
[the remark this answers is not in the document]
Comment (ashu-011) [reply container-images/tcib/base/os/horizontest/run_horizontest.sh]: done

File: container-images/tcib/base/os/horizontest/run_horizontest.sh [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+pushd ${HORIZONTEST_DIR}/horizon/openstack_dashboard/test/integration_tests/
+
+# set variables in horizon.conf
+crudini --set horizon.conf dashboard dashboard_url ${DASHBOARD_URL}/dashboard/ && \
+    crudini --set horizon.conf dashboard auth_url ${AUTH_URL} && \
+    crudini --set horizon.conf identity username ${USER_NAME}  && \
+    crudini --set horizon.conf identity password ${PASSWORD} && \
+    crudini --set horizon.conf identity home_project ${PROJECT_NAME}  && \
+    crudini --set horizon.conf identity admin_username ${ADMIN_USERNAME} && \
+    crudini --set horizon.conf identity admin_password ${ADMIN_PASSWORD}
```
[the remark this answers is not in the document]
Comment (ashu-011) [reply container-images/tcib/base/os/horizontest/run_horizontest.sh]: done , thanks

**Review by lpiwowar — COMMENTED** [review]
Looks good to me but the trailing whitespaces are still present:

```
[lpiwowar@fedora tcib]$ bashate container-images/tcib/base/os/horizontest/run_horizontest.sh 
container-images/tcib/base/os/horizontest/run_horizontest.sh:13:1: E001 Trailing Whitespace
container-images/tcib/base/os/horizontest/run_horizontest.sh:82:1: E001 Trailing Whitespace
container-images/tcib/base/os/horizontest/run_horizontest.sh:106:1: E006 Line too long
container-images/tcib/base/os/horizontest/run_horizontest.sh:106:1: E006 Line too long
```

**Review by lpiwowar — APPROVED** [review]
LGTM! :+1:

**Pull request merged**